### DISCUSSION TITLE
DAI 2024-2025 - Practical work 3 - Mathieu Rabot, Camille Theubet and Antoine Aubry

### DISCUSSION THREAD

**Discussion by CamilleTCH** [heig-vd-dai-course/heig-vd-dai-course#536] [Show and tell]
During this project, we will take the opportunity we're given to mix our BDR long term project with the DAI practical work #3.

For context, during our BDR long term project, we designed a database that is able to store information about the Olympic Games !

For detailed information about what our database can store, see the Cahier des Charges we established during phase 1.
[rapport_phase1_v1_0_1.pdf](https://github.com/user-attachments/files/18124754/rapport_phase1_v1_0_1.pdf)

So what we will do in the DAI part is create an API to abe to add, remove, and modify data in this database.

The CRUD will allow the user to add/delete/modify athletes, teams of athletes, and their respective participation in the avalaible events. Each athlete team that participates at an event (for example, men 100 m run) will first need to register to one of the available editions of the olympic games, for one of the available federations. Editions and federations will be created by us manually for simplicity.

@GeraudSilvestri updated

**Comment by GeraudSilvestri** [comment]
Good for me

**Comment by CamilleTCH** [comment]
Lien vers repo : https://github.com/Nomeos/DAI-2024-2025-Practical-work-3-Mathieu-Rabot-and-Camille-Theubet-and-Antoine-Aubry

Commit rendu: 9a8eb718ab69b8819a3da9b5c08c4593fa7b1714

**Reply by CamilleTCH** [reply]
J'ai bien sué
